package.json

```diff
@@ -0,0 +1,131 @@
+{
+  "name": "terra-site",
+  "version": "0.0.0",
```

Comment:
the release scripts make this interesting, but the initial release of this version of terra site should be 2.0

Reply:
We should update this to `1.0.0` and then we we run the release, use the `release:major` major script